pages/404.tsx

```diff
@@ -1,6 +1,5 @@
 import React from 'react'
-import Error from 'next/error'
-
+import Error from '../components/Error'
 export default function NotFound() {
-  return <Error statusCode={404} />
+  return <Error title="404 error" src="/404.png" message="Page not found" />
```

Comment:
Next.JS has a default 404 page. Please utilize this instead of creating a 404 page.

Reply:
I don't understand, according to docs next.js [suggests](https://nextjs.org/docs/advanced-features/custom-error-page) to edit `pages/404.tsx` to create custom error page. Do you want me to edit next.js error component instead of creating my own?

Reply:
That sounds like a much better way and less confusing with the code. For the error page, you can just render a link to login or the landing page.  It's better if you don't pull any data into it since it's statically generated. 

I like the idea of using next.js's error capability. Since it will go to the page whenever there's an error and you don't have to manually route to `/500`

https://nextjs.org/docs/advanced-features/custom-error-page

Reply:
>Otherwise, users won't be redirected to 404 when they go to c0d3.com/500 for whatever reason.
c0d3.com/500 won't redirect to 404 page, but you're right, I forgot to update _error page. I updated it and it should display new Error screens now too.